docs/blog/newsletter-september-2023.rst

```diff
@@ -0,0 +1,113 @@
+.. post:: September 05, 2023
+   :tags: newsletter
+
+##########################################
+Write the Docs Newsletter – September 2023
+##########################################
+
+Hey, documentarians, the newsletter is back!
+
+In conference news, we have the `2023 Atlantic conference </conf/atlantic/2023/>`__ running Sunday through Tuesday. If you don't have a ticket yet, grab one while you can (through 7 September). Looking forward to seeing you there. And after you get inspired, the Australia conference still has its `call for proposals </conf/australia/2023/cfp/>`__, but it closes 15 September, so get your ideas in soon.
+
+It's almost that time again: the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate, whether you're working or currently unemployed, full-time or part-time, permanent or contract, employee or freelance, in all countries, all industries, and no matter what level of skill or experience you possess. Your contribution will help the whole community better understand what an appropriate salary is, and provide a basis for future negotiations.
+
+This year, we're removing the questions on workplace changes related to the pandemic, and including new questions about "return to the office" mandates, job changes and your thoughts on job security - issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and as always you can find all the details at https://salary-survey.writethedocs.org/.
```

Comment:
I'd prefer a single paragraph, since the newsletter is already rather long as is. So it'd be a little shorter this way.

```suggestion
Elsewhere, the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate. Your contribution will help the whole community better understand what an appropriate salary is and provide a basis for future negotiations. This year, we're removing the questions on workplace changes related to the pandemic and including new questions about "return to the office" mandates, job changes, and your thoughts on job security – issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and you can always find all details at https://salary-survey.writethedocs.org/.
```

Reply:
I'd like to keep the mention of unemployed people and freelance/contract in - how about this?

Elsewhere, the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate - working or currently unemployed, contract or permanent, employee or freelance. Your contribution will help the whole community better understand what an appropriate salary is and provide a basis for future negotiations. This year, we're including new questions about "return to the office" mandates, job changes, and your thoughts on job security – issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and you can always find all details at https://salary-survey.writethedocs.org/.

Reply:
```suggestion
Elsewhere, the Write the Docs Salary Survey for 2023 is opening next week. We encourage everyone working in areas related to documentation to participate - working or currently unemployed, contract or permanent, employee or freelance. Your contribution will help the whole community better understand what an appropriate salary is and provide a basis for future negotiations. This year, we're including new questions about "return to the office" mandates, job changes, and your thoughts on job security – issues at the top of a lot of people's minds. We'll be announcing when the survey goes live on Slack and Twitter, and you can always find all details at https://salary-survey.writethedocs.org/.
```